headless/index.jsx

```diff
@@ -54,9 +54,10 @@ function App() {
 						<li>
 							Or your own design system 💪
 						</li>
-					</ul>"
-				onReady={ ( editor ) => {
-					setEditor( editor );
+					</ul>;
```

Comment:
`;` stays in the editor content:
![image](https://user-images.githubusercontent.com/10883184/229759220-00335986-d6b6-4f36-aabf-53b862d8e7e8.png)